### PR TITLE
Add hass_midea_acc

### PR DESCRIPTION
**Pull request by wyapx**

### CODE REVIEW

**Comment by homeassistant** [comment]
Hi @wyapx, 

It seems you haven't yet signed a CLA. Please do so [here](https://home-assistant.io/developers/cla_sign_start/?pr=home-assistant&#x2F;wheels-custom-integrations#490).

Once you do that we will be able to review and accept this pull request.

Thanks!

File: components/midea_ac_lan [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "name": "Midea AC on LAN",
+  "owner": ["@wyapx"],
+  "manifest": "https://github.com/wyapx/hass_midea_acc/raw/master/custom_components/midea_lan/manifest.json",
```
Comment (ludeeus) [inline components/midea_ac_lan:4]: This link is a redirect, use the target for that URL instead.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: components/midea_ac_lan (hunk 1 above)
Comment (wyapx) [reply components/midea_ac_lan:4]: Okay, I will make change

**Review by ludeeus — APPROVED** [review]
Thanks :+1:

**Pull request merged**